index.yaml

```diff
@@ -28,6 +28,19 @@ entries:
     - https://sematic-ai.github.io/helm-charts/sematic-grafana-dashboards/sematic-grafana-dashboards-1.0.0.tgz
     version: 1.0.0
   sematic-server:
+  - apiVersion: v2
+    appVersion: v0.29.0
+    created: "2023-05-04T04:18:14.423675638Z"
+    description: Sematic AI Server
+    digest: c38bb19bc93336de7631fbf50daf3939e818a9c25c49d662da6f6457333bda00
+    maintainers:
+    - name: sematic-ai
+      url: https://github.com/sematic-ai/sematic/
+    name: sematic-server
+    type: application
+    urls:
+    - https://sematic-ai.github.io/helm-charts/sematic-server/sematic-server-1.1.8.tgz
+    version: 1.1.8
```

Comment:
Please rename the PR to "Release 1.1.8 with Sematic 0.29.0", and add a PR description.